examples/rubyonrails/.test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -ex
+rails new blog -d=postgresql
+devenv up&
+timeout 20 bash -c 'until echo > /dev/tcp/localhost/5100; do sleep 0.5; done'
+pushd blog
+    rails db:create
+popd
```

Comment:
CI is failing for `pushd`. I guess `sh` doesn't support it.

```suggestion
(cd blog && rails db:create)
```